README.md

```diff
@@ -866,7 +866,7 @@ FastRoute has an alternative tool for the routing topology construction, called
 priority of PDRev between wire length and skew, using the `alpha` parameter.
```

Comment:
suggest trade-off between minimum wire length and path length between the driver and the loads